website/integrations/services/apple/index.md

```diff
@@ -0,0 +1,295 @@
+
+| Field                | Value                        |
+| -------------------- | ---------------------------- |
+| Name                 | Apple Business Manager       |
+| Slug                 | `abm`                        |
+| Provider             | Apple Business Manager       |
+| Backchannel Provider | `apple-business-manager-ssf` |
+
+With the application created, you're almost ready to switch over to Apple Business Manager.
+
+:::tip{title="SSF Config URL"}
+
+Assigning an SSF provider to a application will generate _SSF Config URL_ value. You'll need this when configuring Apple Business Manager, so make sure navigate to **Providers** -> **Apple Business Manager SSF** and copy the _SSF Config URL_ value to your text editor.
+
+:::
+
+---
+
+## Apple Business Manager Configuration
+
+Let's recap what you've done to configured authentik to act as an identity provider for Apple Business Manager.
+
+1. You've created three new scope mappings to handle the user's profile information, read access, and management access.
+2. Created a OAuth2/OpenID Provider to handle the authentication flow.
+    - Added the new scope mappings to the provider.
+    - And noted the _Client ID_, _Client Secret_, and _OpenID Configuration URL_.
+3. Created a Shared Signals Framework Provider to handle the backchannel communication between authentik and Apple Business Manager.
+    - Noted the _SSF Config URL_.
+4. Created application configuration for Apple Business Manager.
+
+With these prerequisites in place, authentik is primed to act as an identity provider for Managed Apple Accounts.
+
+:::info{title="Managed Apple Accounts"}
+
+Similar to a user's personal Apple account, a _Managed Apple Account_ is created and managed by an organization, and can be used to access Apple services and devices. Both use an email address as the user name, but a _Managed Apple Account_ is owned by the organization, not the user.
+
+:::
+
+Let's continue our federated authentication setup by signing into the [Apple Business Manager Dashboard](https://business.apple.com/).
+
+### Add and verify your domain
+
+Starting from the Apple Business Manager dashboard, press **your account name** on the sidebar, then select **Preferences**. From the preferences page, select **Managed Apple Accounts** tab, and press **Add Domain**.
+
+After providing your domain name, continue by pressing **Verify**. Apple will generate a DNS TXT record that you'll need to add to your domain's DNS settings. Once you've added the TXT record (and waited for DNS propagation), press **Verify** to complete the process.
+
+#### Lock your domain
+
+Upon successful domain verification, you'll be asked to confirm that you wish to lock your domain. Locking your domain ensures that only your organization can use your domain for federated authentication.
```

Comment:
```suggestion
Upon successful domain verification, you are asked to confirm that you wish to lock your domain. Locking your domain ensures that only your organization can use your domain for federated authentication.
```

Reply:
In procedural docs, we try to use present tense, because the action is actually happening at that moment.